docs/src/modules/components/AppNavDrawer.js

```diff
@@ -166,6 +166,7 @@ function PersistScroll(props) {
     }
 
     return () => {
+      // TODO: uncomment once we enable eslint-plugin-react-compiler // eslint-disable-next-line react-compiler/react-compiler -- useEnhancedEffect uses useEffect under the hood
```

Comment:
What's the plan with these comments? To fix it in the future or to keep the disable?

Reply:
The plan is to keep the disable but remove the comment/TODO once we enable the corresponding ESLint plugin.